OrbitQt/CallTreeWidget.h

```diff
@@ -92,13 +95,17 @@ class CallTreeWidget : public QWidget {
   void SetCallTreeView(std::unique_ptr<CallTreeView> call_tree_view,
                        std::unique_ptr<QIdentityProxyModel> hide_values_proxy_model);
 
+  void ResizeColumnsIfNecessary();
+
   std::unique_ptr<Ui::CallTreeWidget> ui_;
   OrbitApp* app_ = nullptr;
   std::unique_ptr<CallTreeViewItemModel> model_;
   std::unique_ptr<QIdentityProxyModel> hide_values_proxy_model_;
   std::unique_ptr<HighlightCustomFilterSortFilterProxyModel> search_proxy_model_;
   std::unique_ptr<HookedIdentityProxyModel> hooked_proxy_model_;
-  bool columns_already_resized_ = false;
+
+  enum class ColumnResizingState { kInitial = 0, kWidgetSizeSet = 1, kDone = 2 };
```

Comment:
Maybe add explanation for these states here?

Reply:
Added a brief explanation for each state.